openfisca_us/variables/finance/tax/outputs.py

```diff
@@ -998,9 +999,31 @@ class invinc_ec_base(Variable):
     value_type = float
     entity = TaxUnit
     definition_period = YEAR
-    documentation = (
-        """search taxcalc/calcfunctions.py for how calculated and used"""
-    )
+    label = "AGI investment income exclusion"
+    unit = "currency-USD"
+    documentation = """Exclusion of investment income from AGI"""
+
+    def formula(tax_unit, period, parameters):
+        # Limitatition on net short-term and
+        # long-term capital losses
+        limited_capital_gain = max_(
+            -3000.0 / tax_unit("sep", period),
```

Comment:
So...I kind of think we should just parameterize/make a new variable for this, given they're doing it in https://github.com/PSLmodels/Tax-Calculator/issues/2624

Could be a separate PR if you prefer though

Reply:
Yep, filed in [#258](https://github.com/PolicyEngine/openfisca-us/issues/258)